### ISSUE TITLE
Please add my Kingdom to your clan!

### ISSUE THREAD

**Issue by KieranAlexis** [blockstack/animal-kingdom#565]
https://jolly-bohr-1ef536.netlify.com/

kieranalexis.id.blockstack

**Comment by moxiegirl** [comment]
@KieranAlexis Thank you for joining our clan! Your animal kingdom can be found listed in the clan here: https://animalkingdoms.netlify.com/others.  Please don't forget to submit your "Blockstack Sample Application" to https://app.co/ and claim your limited edition Zero-to-Dapp t-shirt.

But who are the Metabarons?

**Comment by KieranAlexis** [comment]
Hi there!

Great news, thanks for letting me know.
I'm super excited to be involved, really like the Blockstack ecosystem,
and am very stoked to be on my way to learning and creating much more.
Plus, I can't wait to get the shirt ;)

So. . . .  I'm glad you asked!
The Metabarons are part of a little known Alejandro Jodorowsky/Moebieus
comic series universe, published mainly in French, called 'The Incal' that I loved in the 80's.

Later on there was a separate Metabarons spin off comic, and they redid the whole original series in English as well. . the image I took was from that same Incal universe . . .and later on in the 90s I released some techno using the name as well . . . lol . . so it just sprang to mind :)

Here's a trailer that got made much later, but the film never happened
https://www.youtube.com/watch?v=c4Vkyzrs1Fk&t=93s

Hope that's not too much info he he

Kind regards,

Kieran.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
